src/common/components/htmlToReact/replaceComponents/DefaultH2.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+import Text from '../../text/Text';
+
+export default function DefaultH2({ children }: { children: React.ReactNode }) {
```

Comment:
DefaultP and H2 looks more or less the same, Maybe just with extra param it can be combined to one. Which might not be the case if we cannot pass params. So its ok in general.